resources/wikia/ui_components/modal/css/modal-close-icon.scss

```diff
@@ -0,0 +1,19 @@
+@import "skins/shared/color";
+@import "skins/shared/mixins/image-replacement";
+
+$close-image: '/resources/wikia/ui_components/modal/images/close-dark.svg';
+
+@if( $is-dark-wiki ) {
+	$close-image: '/resources/wikia/ui_components/modal/images/close-light.svg';
+}
+
+@mixin modal-close-icon() {
+	@include image-replacement();
+	background: url( $close-image );
```

Comment:
Parentheses [should not be padded by spaces](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#spacebetweenparens).